test/verify/check-storage-lvm2

```diff
@@ -957,6 +960,25 @@ class TestStorageLvm2Destructive(storagelib.StorageCase):
         self.dialog({})
         b.wait_not_present(".pf-v6-c-alert")
 
+    @testlib.skipImage("TODO: don't know how to move the rootfs", "debian-*", "ubuntu-*", "arch")
+    # this doesn't work in ostree either, but we don't run storage tests there
+    @testlib.skipWsContainer("newly built root doesn't contain ws container")
+    @testlib.timeout(1200)
+    def testRenameRoot(self):
```

Comment:
This is destructive due to the `modify_rootfs` call presumably. But any chance it could be merged into an existing test? These are quite expensive for such a short test.

Reply:
Yeah, tbh, I don't know why I didn't just put it into the existing test with the root encryption...